src/eradiate/radprops/_us76_approx.py

```diff
@@ -146,17 +146,20 @@ class US76ApproxRadProfile(RadProfile):
         default="True",
     )
 
-    absorption_data_set: str | None = documented(
+    absorption_dataset: PathLike | xr.Dataset | None = documented(
```

Comment:
AFAIK you're not storing the path, event if you allow it in the constructor:
```suggestion
    absorption_dataset: xr.Dataset | None = documented(
```